client/src/components/NewPizzaPlaceForm.js

```diff
@@ -0,0 +1,149 @@
+                }else{
+                    throw new Error(`${response.status} (${response.statusText})`)
+                }
+            } else {
+                const body = await response.json()
+                              
+                setShouldRedirect(true)
+               
+            }
+        } catch(error) {
+            console.error(`Error in fetch: ${error.message}`)
+        }
+    }
+
+
+    const handleInputChange = (event) => {
+        setNewPizzaPlace({
+            ...newPizzaPlace,
+            [event.currentTarget.name]: event.currentTarget.value
+        })
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        postNewPizzaPlace(newPizzaPlace)
+        clearForm()
+    }
+
+    const clearForm = () => {
+        setNewPizzaPlace({
+            name: "",
+            address: "",
+            phoneNumber: "",
+            website: "",
+            hours: "",
+            imageUrl: ""
+        })
+    }
+
+    if(shouldRedirect) {
+        return <Redirect push to="/pizza-places" />
+    }
+
+  return (
+    <form onSubmit={handleSubmit} >
+        <ErrorList errors={errors}/>
+        <label htmlFor='name'>
+            Name:
+            <input 
+                type='text' 
+                name='name'
+                onChange={handleInputChange} 
+                value={newPizzaPlace.name}
+            />
+        </label>
+
+        <label htmlFor='address'>
+            Address:
+            <input 
+                type='text' 
+                name='address'
+                onChange={handleInputChange} 
+                value={newPizzaPlace.address}
+            />
+        </label>
+
+        <label htmlFor='phoneNumber'>
+            Phone Number:
+            <input 
+                type='text' 
+                name='phoneNumber'
+                onChange={handleInputChange} 
+                value={newPizzaPlace.phoneNumber}
+            />
+        </label>
+
+        <label htmlFor='website'>
+            Website Address:
+            <input 
+                type='text' 
+                name='website'
+                onChange={handleInputChange} 
+                value={newPizzaPlace.website}
+            />
+        </label>
+
+        <label htmlFor='hours'>
+            Hours:
+            <input 
+                type='text' 
+                name='hours'
+                onChange={handleInputChange} 
+                value={newPizzaPlace.hours}
+            />
+        </label>
+
+        <label htmlFor='imageUrl'>
+            Add Photo (image url):
+            <input 
+                type='text' 
+                name='imageUrl'
+                onChange={handleInputChange} 
+                value={newPizzaPlace.imageUrl}
+            />
+        </label>
+
+        <div>
+            <input className='button' type='submit' value='Submit' />
+            <input className='button' type='button' value='Clear Form' onClick={clearForm}/>
+        </div>
```

Comment:
this div doesnt do anything. Only add divs if you need a group of elements to style. remove it!